src/main/java/DeleteCommand.java

```diff
@@ -0,0 +1,19 @@
+public class DeleteCommand extends Command {
+    DeleteCommand(int itemNo) {
+        super();
+        super.itemNo = itemNo;
+    }
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+    @Override
+    public String getType() {
+        return "delete";
+    }
+
+    @Override
+    public String getDescription() {
+        return null;
```

Comment:
Might not be a good idea to return null; could potentially result in NullPointerExceptions somewhere down the line.